Pj_MonteVerdi_defender.gmx/objects/obj_platform_Orpheus.object.gmx

```diff
@@ -29,15 +29,19 @@
 
 if(global.stop) exit;
 
-//Check if its tangible
-if(instance_exists(obj_Orpheus)){
-    if(round(obj_Orpheus.y+(obj_Orpheus.sprite_height/2))&lt;=round(y-sprite_height/2)) {mask_index = spr_platform}  //If the players' feet are over the top and it isn`t in the same horizontal space of the platform it becomes solid  
-    else {mask_index = spr_platform_intangible;}                                                                  //Otherwise it makes its collision mask the one of and non-existing sprite            
+//Timer
```

Comment:
I think i would be nice to start moving all the behaviour to scripts instead of just leaving it in the object event